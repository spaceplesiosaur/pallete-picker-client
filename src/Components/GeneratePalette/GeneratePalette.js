import React from 'react';
import './GeneratePalette.scss';

const GeneratePalette = () => {
    return ( 
      <button> Generate Palette !</button>
    )
}

export default GeneratePalette;