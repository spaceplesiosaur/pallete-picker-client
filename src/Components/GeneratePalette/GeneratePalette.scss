.generate-button {
    background: transparent;
    border: 3px solid #B3B3F1;
    padding: 10px;
    height: 8em;
    border-radius: 50%;
    font-size: 1em;
    position: relative;
    top: 14em;
    cursor: pointer;
    &:hover {
      background-color: #B3B3F1;
      font-weight: bold;
    }
}