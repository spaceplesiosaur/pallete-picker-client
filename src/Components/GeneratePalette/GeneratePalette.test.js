import React from 'react';
import { shallow } from 'enzyme';
import GeneratePalette from './GeneratePalette'

describe('GeneratePalette', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<GeneratePalette
      randomizeColors={jest.fn()}
      />)
  })

  it('should render with the correct properites', () => {
    expect(wrapper).toMatchSnapshot();
  })
})
