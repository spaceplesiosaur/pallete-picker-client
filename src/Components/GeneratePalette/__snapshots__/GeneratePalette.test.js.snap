// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GeneratePalette should render with the correct properites 1`] = `
<button
  className="generate-button"
  onClick={[Function]}
>
   Generate Palette !
</button>
`;
